environments/kolla/configuration-ca.yml

```diff
@@ -0,0 +1,2 @@
+kolla_copy_ca_into_containers: "yes"
+openstack_cacert: /etc/ssl/certs/ca-certificates.crt
```

Comment:
environments/kolla is not a group_vars directory. Only the configuration.yml, images.yml + secrets.yml can be used here. You have to add a group_vars directory + yaml files inside if you want to use further files.